### ISSUE TITLE
conda: command not found

### ISSUE THREAD

**Issue by FelipeLujan** [cmdr2/stable-diffusion-ui#81]
Hi, i cloned the repo on a GCP VM and followed the installation steps
 
Im getting the following error when running build.sh (start.sh doesn't exist on the main branch anymore)

logs
**sudo bash ./build.sh**
```
Downloading components for the installer..
./build.sh: line 7: /root/miniconda3/etc/profile.d/conda.sh: No such file or directory
./build.sh: line 9: conda: command not found
./build.sh: line 11: conda: command not found
./build.sh: line 12: conda: command not found
Creating a distributable package..
./build.sh: line 16: conda: command not found
mkdir: cannot create directory ‘installer’: File exists
tar: ../../installer.tar: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
mkdir: cannot create directory ‘scripts’: File exists
Build ready. Zip the 'dist/stable-diffusion-ui' folder.
Cleaning up..
rm: cannot remove 'installer.tar': No such file or directory

**~/stable-diffusion-ui$ uname -a**
Linux instance-2 4.19.0-21-cloud-amd64 #1 SMP Debian 4.19.249-2 (2022-06-30) x86_64 GNU/Linux
```
specs
![image](https://user-images.githubusercontent.com/34170261/188750677-4888b8ea-f28c-4895-99e5-f1c9b7d7178c.png)

**Comment by SharkyGoChompChomp21** [comment]
Hi I'm a Big noob, I left windows & use Linux Mint
I think I have a similar Error with trying to install it on Linux Mint
Here's what I got from the terminal on Linux mint

___________________________________________________________________________________________________
CondaSignalInterrupt: Signal interrupt SIGINT


Error installing the packages necessary for Stable Diffusion. Please try re-running this installer. If it doesn't work, please copy the messages in this window, and ask the community at https://discord.com/invite/u9yhsFmEkB or file an issue at https://github.com/cmdr2/stable-diffusion-ui/issues
___________________________________________________________________________________________________

I'm not sure, like I said a total noob when it comes to linux

**Comment by hortinstein** [comment]
https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html#

install miniconda here...but yeah the update listed no deps...so probably should fix this

**Comment by SharkyGoChompChomp21** [comment]
@hortinstein I'm sorry but I'm just more confused now. shouldn't miniconda download automatically when you run ./start.sh ?
I downloaded [Miniconda3 Linux 64-bit](https://repo.anaconda.com/miniconda/Miniconda3-py39_4.12.0-Linux-x86_64.sh).
I tried to run it in the terminal with "bash: Miniconda3-latest-Linux-x86_64.sh: No such file or directory"
and I get an error. I'm using linux mint, maybe it's a little different?

**Comment by cmdr2** [comment]
@FelipeLujan hi, it looks like you've downloaded the git repository. Please download by using the link in the installation instructions. Thanks!

**Comment by cmdr2** [comment]
@SharkyGoChompChomp21 @hortinstein Developer here - you do not need to install miniconda, it will automatically manage things.

@SharkyGoChompChomp21 Can you please paste the full error message? Also, did you download from the installation instructions or did you download the entire git repository? Thanks!

**Comment by SharkyGoChompChomp21** [comment]
@cmdr2 I deleted the folder I tried to install it from and restarted.
I downloaded it from https://github.com/cmdr2/stable-diffusion-ui#installation
In the installation section I clicked the highlighted "for linux" text and downloaded it.
I extracted the contents to a folder I named StableDiffusion (Specifically with no spaces since I think that was my issue).
I ran it through the linux mint terminal.

And it actually installed, like wow, all I had to do was reinstall it. I was getting some funky errors. but since I deleted the folder I started from, I also deleted the full error message you was asking me for.

Anyways it opened a window in my browser after it finished (Is it supose to do that?)
I typed in a promt but stable diffusion crashed my computer so I had to type all of this up again.
